linters/clang-format/plugin.yaml

```diff
@@ -8,6 +8,11 @@ lint:
         - os: macos
           url: https://trunk.io/releases/clang-format-${version}-macos-x86_64.tar.gz
           version: <14.0.0
+        - os:
+            linux: linux
+          cpu:
+            arm_64: aarch64
+          url: https://trunk.io/releases/clang-format-${version}-${os}-${cpu}.tar.gz
```

Comment:
Can you modify this to make this and the following block be "correct" (more than just first-match)?

Reply:
Done